cea/demand/thermal_loads.py

```diff
@@ -7,6 +7,7 @@
 """
 from __future__ import division
```

Comment:
check that most new functions in this script are lacking a description of input and output parameters